src/packagedcode/phpcomposer.py

```diff
@@ -135,23 +146,30 @@ def is_phpcomposer_json(location):
     return (filetype.is_file(location) and fileutils.file_name(location).lower() == 'composer.json')
 
 
+def is_phpcomposer_lock(location):
+    return (filetype.is_file(location) and fileutils.file_name(location).lower() == 'composer.lock')
```

Comment:
nit picking: The parens wrapping is not needed

Reply:
ack